ros2component/test/test_cli_load.py

```diff
@@ -0,0 +1,217 @@
+                    "[INFO] [talker]: Publishing: 'Hello World'"
+                ], strict=False
+            ), timeout=10)
+        assert load_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=[
+                "Loaded component 1 into '/ComponentManager' "
+                "container node as '/talker'"],
+            text=load_command.output,
+            strict=False
+        )
+        with self.launch_component_command(
+                arguments=[
+                    'load', '/ComponentManager',
+                    'ros2component_test_fixtures',
+                    'ros2component_test_fixtures::Listener']) as load_command:
+            assert load_command.wait_for_shutdown(timeout=20)
+        assert load_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=[
+                "Loaded component 2 into '/ComponentManager' "
+                "container node as '/listener'"],
+            text=load_command.output,
+            strict=False
+        )
+        with self.launch_component_command(
+                arguments=[
+                    'unload', '/ComponentManager', '1']) as unload_command:
+            assert unload_command.wait_for_shutdown(timeout=20)
+        assert unload_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=["Unloaded component 1 from '/ComponentManager' container"],
+            text=unload_command.output,
+            strict=False
+        )
+        # Test the unique id for loaded nodes.
+        with self.launch_component_command(
+                arguments=[
+                    'load', '/ComponentManager',
+                    'ros2component_test_fixtures',
+                    'ros2component_test_fixtures::Talker']) as load_command:
+            assert load_command.wait_for_shutdown(timeout=20)
+        assert load_command.exit_code == launch_testing.asserts.EXIT_OK
+        assert launch_testing.tools.expect_output(
+            expected_lines=[
+                "Loaded component 3 into '/ComponentManager' "
+                "container node as '/talker'"],
+            text=load_command.output,
+            strict=False
+        )
+        # Test we can load the same node more than once.
+        with self.launch_component_command(
+                arguments=[
+                    'load', '/ComponentManager',
+                    'ros2component_test_fixtures',
+                    'ros2component_test_fixtures::Talker']) as load_command:
+            assert load_command.wait_for_shutdown(timeout=20)
+            assert self.component_node.wait_for_output(functools.partial(
+                launch_testing.tools.expect_output, expected_lines=[
+                    "[INFO] [talker]: Publishing: 'Hello World'"
```

Comment:
@BMarchi same about checking container output, not the component's

Reply:
The component node is the container

Reply:
I mean, check the component container's node output, not the talker node's output.